doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -323,22 +338,42 @@ Balance as much as you can
 Try to use the network's card balancing as much as possible ::
  
  for proto in tcp4 udp4 ah4 esp4 sctp4 tcp6 udp6 ah6 esp6 sctp6; do 
- 	/sbin/ethtool -N eth3 rx-flow-hash $proto sdfn
+    /sbin/ethtool -N eth3 rx-flow-hash $proto sd
  done
 
 The XDP CPU redirect case
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If ever your hardware is not able to do a symmetric load balancing but support XDP in driver mode, you
-can then use the CPU redirect map support available in the xdp_filter.bpf file. In this mode, the load
-balancing will be done by the XDP filter and each CPU will handle the whole packet treatment including
-the creation of the skb structure in kernel.
+can then use the CPU redirect map support available in the `xdp_filter.bpf` and `xdp_lb.bpf` file. In 
+this mode, the load balancing will be done by the XDP filter and each CPU will handle the whole packet
+treatment including the creation of the skb structure in kernel.
 
 You will need Linux 4.15 or newer to use that feature.
 
 To do so set the `xdp-cpu-redirect` variable in af-packet interface configuration to a set of CPUs.
 Then use the `cluster_cpu` as load balancing function. You will also need to set the affinity
-accordingly.
+to be sure CPU that will be assigned skb are used by Suricata.
+
+Also to avoid out of order packets, you need to set the RSS queue number to 1. So if our interface
+is `eth3` ::
+
+  /sbin/ethtool -L eth3 combined 1
```

Comment:
is this really necessary as this might be a bottleneck to use just one queue on the NIC side?

Reply:
This could be used to mitigate situations where symmetric hashing is not possible and the only option is to force one queue on the NIC. `xdp-cpu-redirect` will handle the load balancing as explained in the paragraph before.